src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -196,15 +196,30 @@ private async Task<LanguageServerOutputBuilder> OnDataReceivedAsyncInternal(Lang
                 await handler.HandleAsync(context, cancellationToken).ConfigureAwait(false);
                 return outputBuilder;
             }
+            catch (OperationCanceledException operationCanceledEx)
+            {
+                return HandleException(outputBuilder, input, cancellationToken, operationCanceledEx);
+            }

```

Comment:
Does this block do anything?
It looks like a copy of lines 203..206, which is what we'd execute anyways if this block didn't exist.

Reply:
oops, i didn't realize i left a dup block

Reply:
Addressed